test/test.js

```diff
@@ -619,6 +619,14 @@ describe('Page', function() {
       expect(error.message).toContain('Navigation Timeout Exceeded: 1ms');
       process.removeListener('unhandledRejection', unhandledRejectionHandler);
     }));
+    it('should disable timeout when its set to 0', SX(async function() {
+      server.setRoute('/empty.html', (req, res) => {});
+      let error = null;
+      page.goto(PREFIX + '/empty.html', {timeout: 0}).catch(e => error = e);
+      // const sleep = e => new Promise(ff => setTimeout(ff, e));
+      // await sleep(90000);
```

Comment:
> Developer's time is precious! :)

agree, sorry but leave it as comment. Perhaps someone want to inspect this.

Reply:
Please, drop these comments: they are not useful for other puppeteer developers.